### PR TITLE
[MPS] Adding the checks for Convolution and removing FFT from fallback

### PR DESCRIPTION
**Pull request by kulinseth**
Fixes #ISSUE_NUMBER

### CODE REVIEW

File: aten/src/ATen/native/mps/operations/Convolution.mm [hunk 1]
```diff
@@ -66,6 +66,7 @@ Tensor _mps_convolution_impl(
     int64_t groups,
     c10::optional<IntArrayRef> input_shape) {
   TORCH_CHECK(input_t.dim() < 5, "Conv3D is not supported on MPS");
+  TORCH_CHECK(isFloatingType(input_t.scalar_type()), "Convolution is supported only for Floating types");
```
Comment (DenisVieriu97) [inline aten/src/ATen/native/mps/operations/Convolution.mm:69]: I think we support only FP32

Comment (kulinseth) [reply aten/src/ATen/native/mps/operations/Convolution.mm:69]: No, we do support fp16 and we will be supporting bfloat..

Comment (DenisVieriu97) [reply aten/src/ATen/native/mps/operations/Convolution.mm:69]: my bad, FP16 is also supported

**Review by DenisVieriu97 — APPROVED** [review]
LGTM

**Pull request merged**